### PR TITLE
Update rails configuration files

### PR DESCRIPTION
**Pull request by theharq**
#### What this PR does:

This PR updates libv8 minor version to be compatible with latest version of `command-line-tools` for `El Capitan` https://github.com/cowboyd/libv8/compare/v3.16.14.7...v3.16.14.13

I've notice that the configuration files from rails where not updated.

Everytime we update the `rails` gem is a good idea to run the `rake rails:update` rake task in order to have the latest config files and make easier to update to new versions of rails in the future.

### CODE REVIEW

**Comment by theharq** [comment]
PR re-openned, now poiting to the `cf` branch which is now up to date with `master`

File: Gemfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ gem "curb", "~> 0.8"
 
 gem "nokogiri", "~> 1.6"
 
-gem "rails_12factor"
+gem "rails_12factor", group: :production
```
Comment (matthewford) [inline Gemfile]: We would then need to run staging, in ENV production, which is fine, but need to differentiate by perhaps domain (as we do in other apps)

File: config/secrets.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+development:
+  secret_key_base: a9bca5cbc2c7719c10133623bbe336851fb70269cf5cbf2a90b677ee80d3e68ba6f8fd0b5f379fa73f4aa1729c556a1e0cb211ac47f804ab8813c0519578ca10
+
+test:
+  secret_key_base: d07b10fa2256be6c9c2f6dcbd85061d8adb6623889e0ddf3e1651fa113c19022e659ead26f0fb6a5a767c7621085965d00c1b0d5291f37ee7668e893b50a2eac
+
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
+production:
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
```
Comment (matthewford) [inline config/secrets.yml]: We should probably maintain a list somewhere of .env variables we need to set .env.example perhaps?

File: Gemfile (hunk 1 above)
Comment (theharq) [inline Gemfile]: yeah, if we are following the http://12factor.net/config we should stick with the 3 `enviroments` from Rails, (production, development and tests) and store the config in environment variables so we can have multiple apps with the rails production enviroment.

File: config/secrets.yml (hunk 2 above)
Comment (theharq) [inline config/secrets.yml]: I like this idea.. goes in the same direction of the 12 factors app :smiley:  and we can use something like `dotenv` or docker in the development mode

File: Gemfile (hunk 1 above)
Comment (matthewford) [inline Gemfile]: well I prefer to run staging and production in the same env, and just use different env vars for db's etc

File: config/secrets.yml (hunk 2 above)
Comment (matthewford) [inline config/secrets.yml]: if you look in the branches, think @macool started this

Comment (theharq) [inline config/secrets.yml]: :ok_hand:  perfect I'll check his branches

**Pull request merged**